engine/src/test/java/org/hibernate/validator/test/internal/engine/valuehandling/ContainerElementPropertyPathTest.java

```diff
@@ -54,7 +55,6 @@ public void testMapInvalidKeyTypeArgument() {
 
 		Set<ConstraintViolation<DemographicStatistics>> constraintViolations = validator.validate( statictics );
 
-		assertCorrectPropertyPaths( constraintViolations, "inhabitantsPerAddress<K>[].<map key>" ); // the key is null, thus the '[]'
```

Comment:
Not really. We check the path and the string representation. Reason why I divided the tests in 2 parts: the path part can go in the TCK, the string representation part is implementation specific.